src/inmanta/data/model.py

```diff
@@ -786,6 +786,16 @@ class CurrentUser(BaseModel):
     username: str
 
 
+class Role(BaseModel):
```

Comment:
I was a bit confused here, because in the DB roles are global, here they are per env.

But from the code, I can derive this is an assignment of roles to a user.

I wonder if we should either add the user as a field or change the name to `RoleAssignment`. 

I'm uncertain, so feel free to ignore.

Reply:
I went for the rename to `RoleAssignment`.